### PR TITLE
Filter domain with dns challenge

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Review by yann-soubeyrand — COMMENTED** [review]
I think the selector belongs to the same block than dns01.

File: modules/aks/azure/values.tmpl.yaml [hunk 1]
```diff
@@ -75,6 +75,9 @@ cert-manager:
       enabled: true
     acme:
       solvers:
+        - selector:
```
Comment (yann-soubeyrand) [inline modules/aks/azure/values.tmpl.yaml:78]: I think the selector belongs to the same block than dns01.

Comment (Vampouille) [reply modules/aks/azure/values.tmpl.yaml]: Fixed by d8291cc

**Review by yann-soubeyrand — APPROVED** [review]

**Comment by raphink** [comment]
What if we want to request a certificate with multiple domain names (all managed on Azure)?

**Comment by yann-soubeyrand** [comment]
> What if we want to request a certificate with multiple domain names (all managed on Azure)?

IIRC, the challenge used for each domain name depends on the solvers selector. So, if we add an HTTP solver without selector, certificate requests for the base domain will use the DNS challenge and all other requests will use the HTTP challenge. In projects where we want to use DNS challenge for other domains, we should add new solvers.

**Review by mcanevet — APPROVED** [review]

**Pull request merged**